src/ContactModels/include/BipedalLocomotionControllers/ContactModels/ContinuousContactModel.h

```diff
@@ -0,0 +1,141 @@
+/**
+ * @file ContinuousContactModel.h
+ * @authors Giulio Romualdi
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_CONTROLLERS_CONTACT_MODELS_CONTINUOUS_CONTACT_MODEL_H
+#define BIPEDAL_LOCOMOTION_CONTROLLERS_CONTACT_MODELS_CONTINUOUS_CONTACT_MODEL_H
+
+#include <iDynTree/Core/Transform.h>
+#include <iDynTree/Core/Twist.h>
+#include <iDynTree/Core/Wrench.h>
+
+#include <BipedalLocomotionControllers/ContactModels/ContactModel.h>
+
+namespace BipedalLocomotionControllers
+{
+namespace ContactModels
+{
+/**
+ * ContinuousContactModel is a model that describe the contact using a continuous representation. It
+ * is an extension of the brush model used for describing the contact between the tire and the
+ * ground. Each point in the contact surface is subjected to an infinitesimal force \f$ f \f$ given
+ * by \f$ f=k(x_0 - x)- b x\f$, where \f$k\f$ is the spring coefficient and \f$b\f$ is the damper
+ * coefficient. \f$x\f$ and \f$x_0\f$ are, respectively, the position of the point placed in the
+ * contact surface and the point con corresponding to a null contact force written in the inertial
+ * frame. Furthermore, during the contact scenario, we assume the link acts as a rigid body. While
+ * the environment will deform. The ground characteristics are isotropic, and it can be approximated
+ * as a continuum of springs and dampers.
+ * The contact surface is supposed to be <b>rectangular</b> and the frame placed on the contact
+ * surfaced is centered in the middle of the surface, with the \a z-axis pointing upwards and the \a
+ * x-axis pointing forward (parallel to one edge of the rectangle).
```

Comment:
Maybe you can define here what do you mean by length and width. In addition, I would mention that this model assumes infinite friction.